test/evergreen.yml

```diff
@@ -1452,12 +1458,20 @@ variables:
       num_jobs: $(echo $(sysctl -n hw.logicalcpu) / 2 | bc)
       cmake_generator: "Unix Makefiles"
       additional_env_vars: |
+        export HOMEBREW_PREFIX="$(brew --prefix)"
+        export LLVM_PREFIX="$HOMEBREW_PREFIX/opt/llvm"
```

Comment:
There has been a history of problems where macos tools are not consistently available on all machines. Can we double check if this is consistently avaliable?

Reply:
Yes, I asked DevProd team what is the right way to use different compiler. They say that LLVM is available on their macOS hosts. This is the official way.